[2025-07-04 19:36:45,148] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.201.6:5000 - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:36:45,149] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:37:09,736] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.201.6:5000 - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:37:09,737] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:43:48,078] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:43:48] "GET /flights HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:43:48,082] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:43:48] "GET /airlines HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:44:33,105] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:44:33] "GET /flights?nome=teste HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:44:33,266] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:44:33] "POST /flight HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:44:33,280] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:44:33] "GET /flights HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:44:33,282] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:44:33] "GET /airlines HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 19:44:35,260] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 19:44:35] "GET /airline?index=8 HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:43,295] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.201.6:5000 - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:43,296] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:58,831] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:16:58] "GET /airlines HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:58,833] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:16:58] "GET /flights HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:58,928] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:16:58] "GET /airline?index=8 HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:58,930] ERROR log_exception() L1414 Exception on /tails [GET] - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py L1414
Traceback (most recent call last):
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask_openapi3/scaffold.py", line 127, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/api/app.py", line 79, in get_tails
    return apresenta_tails(tails), 200
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/api/schemas/tail_schema.py", line 43, in apresenta_tails
    "airline": tail.tail
               ^^^^^^^^^
AttributeError: 'Tail' object has no attribute 'tail'
[2025-07-04 21:16:58,935] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:16:58] "[35m[1mGET /tails HTTP/1.1[0m" 500 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:16:58,938] ERROR log_exception() L1414 Exception on /tail [GET] - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py L1414
Traceback (most recent call last):
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/flask_openapi3/scaffold.py", line 127, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/api/app.py", line 297, in get_tail
    return apresenta_tail(tail), 200
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/takimoto/MVP2/flight_delay/api/schemas/tail_schema.py", line 30, in apresenta_tail
    "tail": tail.tail
            ^^^^^^^^^
AttributeError: 'Tail' object has no attribute 'tail'
[2025-07-04 21:16:58,939] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:16:58] "[35m[1mGET /tail?index=54 HTTP/1.1[0m" 500 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:54,462] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.201.6:5000 - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:54,462] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:58,239] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:20:58] "GET /flights HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:58,242] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:20:58] "GET /airlines HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:58,343] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:20:58] "GET /airline?index=8 HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:58,352] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:20:58] "GET /tails HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
[2025-07-04 21:20:58,362] INFO _log() L97   127.0.0.1 - - [04/Jul/2025 21:20:58] "GET /tail?index=54 HTTP/1.1" 200 - - call_trace=/home/takimoto/MVP2/flight_delay/back/lib/python3.12/site-packages/werkzeug/_internal.py L97  
